lib/register/format.js

```diff
@@ -11,6 +11,9 @@
  * and limitations under the License.
  */
 
+ var fs = require('fs'),
+     _  = require('lodash'),
+     chalk = require('chalk');
```

Comment:
we're not using `chalk` in the code below, right?

Reply:
Good catch

Reply:
Not using any of those, in fact. Migrated the dependencies from template.js but didn't end up needing them.